submissions/AlexeiGladkov/Hooli-style-popup/index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="shortcut icon" href="image/favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <header>
+        <div class="conteiner">
+            <nav class="nav-popup">
+                <ul class="nav-popup__list">
+                    <li class="nav-popup__list-item">
+                        <a class="nav-popup__link-text" href="#">Mail</a>
+                    </li>
+                    <li class="nav-popup__list-item">
+                        <a class="nav-popup__link-text" href="#">Images</a>
+                    </li>
+                    <li class="nav-popup__list-item nav-popup__list-item-btn">
+                        <input class="nav-popup__checkbox-input" type="checkbox" name="nav-popup__checkbox"
+                            id="nav-popup__checkbox">
+                        <label class="nav-popup__checkbox-label" for="nav-popup__checkbox">
+                            <img class="nav-popup__img-btn" src="image/popup-button.png" alt="buttom">
+                        </label>
+                        <div class="nav-popup__menu">
```

Comment:
Fix popup position on the desktop view
![image](https://user-images.githubusercontent.com/16196199/185844352-9dbb9cd6-a347-4787-9aee-e2cd9df8789d.png)